city_scrapers/spiders/chi_ssa_69.py

```diff
@@ -0,0 +1,130 @@
+import json
+import re
+from datetime import datetime, timedelta
+
+from city_scrapers_core.constants import COMMISSION, COMMITTEE, FORUM, NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiSsa69Spider(CityScrapersSpider):
+    name = "chi_ssa_69"
+    agency = "Chicago Special Service Area #69 95th & Ashland Avenue"
+    timezone = "America/Chicago"
+
+    @property
+    def start_urls(self):
+        today = datetime.now()
+        last_week = today - timedelta(days=7)
+        in_two_months = today + timedelta(days=60)
+
+        # I think there is a problem where the timezone in Chicago will be
+        # changing during tis two month period
+        # This results in meeting times that look wrong (off by an hour)
+
+        from dateutil import tz
+
+        # determine the current GMT offset for Chicago time (varies throughout the year)
+        current_chicago_time = datetime.now(tz.gettz('America/Chicago'))
+        current_chicago_time_gmt_offset = str(current_chicago_time)[-6:]
+
+        return [(
+            "https://www.googleapis.com/calendar/v3/calendars/gagdcchicago%40gmail.com/events"
+            "?calendarId=gagdcchicago@gmail.com&singleEvents=true&timeZone={}&"
+            "sanitizeHtml=true&timeMin={}T00:00:00{}&timeMax={}T00:00:00{}&"
+            "key=AIzaSyC-KzxSLmmZitsCVv2DeueeUxoVwP0raVk"
+        ).format(
+            current_chicago_time_gmt_offset,
+            last_week.strftime("%Y-%m-%d"), current_chicago_time_gmt_offset,
+            in_two_months.strftime("%Y-%m-%d"), current_chicago_time_gmt_offset
+        )]
+
+    def parse(self, response):
+        data = json.loads(response.text)
+
+        for item in data["items"]:
+            title = self._parse_title(item)
+            location = self._parse_location(item)
+            if not location:
+                continue
+            meeting = Meeting(
+                title=title,
+                description=self._parse_description(item),
+                classification=self._parse_classification(title),
+                start=self._parse_dt(item["start"]),
+                end=self._parse_dt(item["end"]),
+                time_notes="",
+                all_day=self._parse_all_day(item),
+                location=location,
+                links=[],
+                source=item["htmlLink"],
+            )
+            meeting['status'] = self._get_status(meeting, text=item["status"])
+            meeting['id'] = self._get_id(meeting)
+            yield meeting
+
+    def _parse_title(self, item):
+        return re.sub(r" Meeting$", "", item["summary"].strip())
```

Comment:
It looks like they're including police beat meetings on here, so we should ignore anything that includes "CPD" or "C.A.P.S" in the name (we're tracking those separately). There are also some festivals and holidays, so we should check that "Meeting" or "SSA" (to be safe) is in the title and ignore events missing either of those words.

We can also just do a simple `.replace("Meeting", "")` instead of using regex since it doesn't seem necessary here

Reply:
done